DPPP/MSReader.cc

```diff
@@ -183,7 +185,7 @@ namespace DP3 {
         // Round specified first time to a multiple of itsTimeInterval
         itsFirstTime = startTimeMS + std::ceil((startTimeParset - startTimeMS) / itsTimeInterval) * itsTimeInterval;
```

Comment:
In this case you should probably report an error when startTimeSlot != 0 (specifying a time + a startslot is not allowed)